### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Jleev77**
TEST

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# MinecraftCopy
\ No newline at end of file
+# MinecraftCopy
+
+Change
```
Comment (l33753ph1r07h) [inline README.md:3]: This is fucking stupid why would you do this

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# MinecraftCopy
\ No newline at end of file
+# MinecraftCopy
```
Comment (l33753ph1r07h) [inline README.md:1]: This is good though

File: README.md (hunk 1 above)
Comment (Jleev77) [reply README.md:3]: UR FUCKING STUPID

**Review by l33753ph1r07h — APPROVED** [review]
looks good

**Pull request merged**